index.html

```diff
@@ -13,5 +13,49 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Acme Web Sulutions</title>
   </head>
-  <body></body>
+  <body>
+    <header class="site-header">
+      <div class="site-wrapper">
+        <div>header</div>
+      </div>
+    </header>
+    <section class="first-section">
+      <div class="site-wrapper">
+        <div class="first-section-container">
+          <div class="first-section-item">1</div>
+          <div class="first-section-item">2</div>
+        </div>
+      </div>
+    </section>
+    <section class="second-section">
+      <div class="site-wrapper">
+        <ul class="column-container">
+          <li class="column-item">1</li>
+          <li class="column-item">2</li>
+          <li class="column-item">3</li>
+        </ul>
+      </div>
+    </section>
+    <div class="site-wrapper">
+      <section class="third-section">1</section>
+    </div>
+    <div class="site-wrapper">
+      <section class="fourth-section">
```

Comment:
I think it would be better semantically if each item be a section as each of them has a title not rounded by a section.

Reply:
Done.